test/runtests.jl

```diff
@@ -6,3 +6,4 @@ using Base.Test
 include("roots.jl")
 include("newton1d.jl")
 include("quadratic.jl")
+#include("test_smiley.jl")
```

Comment:
Why not running the tests?

Reply:
@gwater commented that one of the tests takes a very long time (40 mins). Maybe we can reduce the tolerance to get it to run faster?

Reply:
I did some tests in my laptop (using IntervalArithmetics.jl v0.13.0) and the last tests throws an error; the time it takes is quite reasonable. The error is related with the actual number of roots obtained (20 instead of 41).

I propose to have the first three tests being checked, and leave the last one commented, for the time being.

Incidentally, I also checked this against current master of IntervalArithmetics (branching from lb/iss78 in IntervalRootFinding); again the first three tests run fine and reasonably fast, but the last one throws a different error.

Reply:
To be more explicit, the test that was not passing is Example 5.5.

Reply:
Hmm, I cannot reproduce the error in Example 5.5. All tests pass on my machine. Some details:
```
julia> versioninfo()
Julia Version 0.6.3
Commit d55cadc350 (2018-05-28 20:20 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, broadwell)

julia> Pkg.status("IntervalArithmetic")
 - IntervalArithmetic            0.13.0
```
I've also run the tests with various tolerances: Tests pass up until a tolerance of `1e-6` above that roots start to get miscategorized or missed. At that tolerance Example 5.5 takes ca. 3 minutes on my machine which may be acceptable for the standard testing suite (esp. considering things will probably speed up more in the future).

So that's my recommendation: Raise tolerance to `1e-6` and add all examples, including Example 5.5, to the `runtests.jl` suite.

Reply:
Apparently Example 5.5 fails on Travis CI, too. To be honest though, hunting this bug seems outside the scope of this pull request to me. (Clearly, the tests are uncovering an issue here but that does not mean they are broken or wrong.)